WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -549,19 +549,10 @@ def __getCEName( self ):
             self.log.error( "CE Name %s not accepted" % CE )
             self.exitWithError( retCode )
         else:
-          self.log.info( "Looking if queue name is already present in local cfg" )
-          from DIRAC import gConfig
-          ceName = gConfig.getValue( 'LocalSite/GridCE', '' )
-          ceQueue = gConfig.getValue( 'LocalSite/CEQueue', '' )
-          if ceName and ceQueue:
-            self.log.debug( "Found CE %s, queue %s" % ( ceName, ceQueue ) )
-            self.pp.ceName = ceName
-            self.pp.queueName = ceQueue
-          else:
-            self.log.error( "Can't find ceName nor queue... have to fail!" )
-            sys.exit( 1 )
+          self.log.error( "Can't find ceName nor queue... have to fail!" )
```

Comment:
Does this change rely on the changes in #2673 ? Should these PRs come together ?